compound_disease/compound_treats_disease/transfer_learning_experiment/transfer_learning.py

```diff
@@ -0,0 +1,270 @@
+
+# coding: utf-8
+
+# # Transfer Learning for Compound Treats Disease Relationship
+
+# Based on previous experiments, the next step is to answer the question: Can label functions transfer between realtionship types? This notebook is designed to answer this question by testing the hypothesis: **Compound Binds Gene (CbG) label functions transfer better than Disease Associates Gene (DaG) functions when classifiying Compound Treats Disease sentences.** To test this hypothesis, we plan to train multiple generative models and measure each models performance in the form of Receiver Operative Cruves (ROC) and Precision-Recall Curves (PRCs) 
+
+# ## Set up the Environment
+
+# In[ ]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+from collections import defaultdict
+import os
+import sys
+
+sys.path.append(os.path.abspath('../../../modules'))
+sys.path.append(os.path.abspath('data/label_functions/'))
+
+import pandas as pd
+from tqdm import tqdm_notebook
+
+
+# In[ ]:
+
+
+#Set up the environment
+username = "danich1"
+password = "snorkel"
+dbname = "pubmeddb"
+
+#Path subject to change for different os
+database_str = "postgresql+psycopg2://{}:{}@/{}?host=/var/run/postgresql".format(username, password, dbname)
+os.environ['SNORKELDB'] = database_str
+
+from snorkel import SnorkelSession
+session = SnorkelSession()
+
+
+# In[ ]:
+
+
+from snorkel.annotations import LabelAnnotator
+from snorkel.models import candidate_subclass
+
+from metal.label_model import LabelModel
+
+from compound_disease_lf import CD_LFS
+from disease_gene_lf import DG_LFS
+from compound_gene_lf import CG_LFS
+from utils.notebook_utils.dataframe_helper import load_candidate_dataframes
+from utils.notebook_utils.label_matrix_helper import (
+    get_columns, label_candidates, get_auc_significant_stats
```

Comment:
(for my personal growth as a programmer) Are you importing python NB here?  I didn't realize you could do this?

Reply:
Not sure you can import notebooks, but in this case I'm just importing scripts that placed in a folder called notebook_utils.